client/state/analytics/middleware.js

```diff
@@ -1,24 +1,43 @@
-import analytics from 'lib/analytics';
-import has from 'lodash/has';
-import invoke from 'lodash/invoke';
+/**
+ * External dependencies
+ */
+import {
+	has,
+	invoke,
+} from 'lodash';
 
+/**
+ * Internal dependencies
+ */
+import {
+	ga,
+	mc,
+	tracks,
+	pageView,
+} from 'lib/analytics';
+import {
+	trackCustomAdWordsRemarketingEvent,
+	trackCustomFacebookConversionEvent,
+} from 'lib/analytics/ad-tracking';
```

Comment:
the idea I originally had would be for `lib/analytics` to reexport these so we could keep them all imported from the same place but logically separate in the code

```js
export {
	trackCustomAdWordsRemarketingEvent as gaw,
	trackCustomFacebookConversionEvent as fb,
} from './ad-tracking'
```

given the existing naming terseness we might consider shortening those names